### ISSUE TITLE
Can't set breakpoints when deploying on HoloLens

### ISSUE THREAD

**Issue by eluchsinger** [Microsoft/MixedRealityToolkit-Unity#1410]
## Overview
When I set a breakpoint in the code and I remote deploy on the HoloLens, the breakpoints get disabled. This worked some time ago, maybe it broke after I updated to Unity 2017.2 or to Windows SDK 10.0.16299.0.

Build is set to Debug and x86 and Windows SDK 10.0.16299.0.

Maybe related to #1398 

## Expected Behavior
Breakpoints should work. The code should stop and be debugabble.

## Actual Behavior
Breakpoints don't get hit. They are disabled when debugging.

## Unity Editor Version
2017.2.0.p1-MRTP4
## Mixed Reality Toolkit Release Version
Probably around 22ade4fa7a1ab4082ab1dd7f5e01b55dca83a5a7

**Comment by eluchsinger** [comment]
@StephenHodgson could you reproduce this bug?

**Comment by StephenHodgson** [comment]
No I cannot

**Comment by eluchsinger** [comment]
It's weird. I can't reproduce it myself now...

**Comment by Taghazog** [comment]
I Have actually the same problem.
I was debugging a Visual Studio project built from Unity, deployed on HoloLens. 
Now, with Unity 2017.2, SDK 10.0.16299, and Visual Studio 2017, breakpoints does not work anymore at all.
I completely reinstalled a fresh Windows 10 pro with version 1709 and it doesn't change, breakpoints just won't work, there is then clearly something wrong either from Unity or Visual or SDK which prevent the app from breaking correctly in Visual Studio with this configuration at the moment (symbols are loaded for Assembly-CSharp).

I didn't find out anything relevant on internet nor myself about why it fails to break but if it doesn't fail at all to break for you guys i would love to know how!

**Comment by genereddick** [comment]
I'm having similar issues but the behavior is inconsistent. 

Breakpoints that are set on Unity functions like Start and Awake are not hit and when running the breakpoint is deleted. Breakpoints on my functions are hit (though the breakpoint is moved to the opening line of the function), but only in the first scene in a multi-scene project.

**Comment by ameijers** [comment]
We got the same problem. We added the last version of the Mixed Reality Toolkit (2017.2.0.p1-MRTP4) to our project and for some reason we were not able to debug anymore.

the last day i have been trying out different things and found out the following:

Tools
Visual Studio 2017 version 15.4.5
Unity 2017.2 or Unity 2017.3 beta
.NET SDK 10.0.166299.0 
Windows 10 with minimum FCU

Created an Unity project with gameobject, script and some simple code
Build VS solution and did debug (debug, x85, remote machine) to HoloLens
No problems. All breakpoints work

Added the Mixed Reality Toolkit as package to Unity solution
Build VS solution and did debug (debug, x85, remote machine) to HoloLens
None of the breakpoints worked anymore.

Removed from the assets folder both folders regarding Sharing (Sharing and SharingWithUNET)
Build VS solution and did debug (debug, x85, remote machine) to HoloLens
No problems. All breakpoints work

Adding the folders back will cause again problems with breakpoints. I expect that there is something in the folders causing problems with the PDB files generated and code debugging in VS.

Hopefully this could help investigate further the solution
